FIX: 에러헨들링 수정

soogineer opened the pull request:
On branch feature/26
 Changes to be committed:
	modified:   src/http/http.controller.js
	modified:   src/http/http.service.js
	modified:   src/routes/http.route.js
	modified:   src/routes/index.route.js
	modified:   yarn.lock

soogineer committed:
FIX: 에러헨들링 수정

 On branch feature/26
 Changes to be committed:
	modified:   src/http/http.controller.js
	modified:   src/http/http.service.js
	modified:   src/routes/http.route.js
	modified:   src/routes/index.route.js
	modified:   yarn.lock

jn33-dev commented on src/http/http.service.js:
```diff
@@ -12,5 +14,13 @@ class HttpService {
+  getDetailPage = async (pageId) => {
+    const getPage = await this.httpRepository.findPage();
+    if (!getPage[pageId]) 
+        throw new ValidationError('해당 게시물을 찾을 수 없습니다.', 412);
```
status code 404로 변경해주세요

jn33-dev commented on src/http/http.controller.js:
```diff
@@ -1,17 +1,30 @@
   pageList = async (req, res, next) => {
     try {
       const pages = await this.httpService.findPageList();
-      if (!pages) throw new ValidationError('페이지가 없습니다.', 404);
```
죄송합니다ㅠㅠ
다시 이 줄 추가해주세요

soogineer committed:
FIX: [PR 리뷰 반영]
    Changes to be committed:
	modified:   src/http/http.controller.js
	modified:   src/http/http.service.js
